src/Altinn.App.Api/Controllers/LookupOrganisationController.cs

```diff
@@ -16,19 +16,19 @@ namespace Altinn.App.Api.Controllers;
 [Consumes(MediaTypeNames.Application.Json)]
 [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
 [Route("{org}/{app}/api/v1/organisations")]
-public class OrganisationSearchController : ControllerBase
+public class LookupOrganisationController : ControllerBase
 {
     private readonly IOrganizationClient _organisationClient;
-    private readonly ILogger<OrganisationSearchController> _logger;
+    private readonly ILogger<LookupOrganisationController> _logger;
 
     /// <summary>
-    /// Initialize a new instance of <see cref="OrganisationSearchController"/> with the given services.
+    /// Initialize a new instance of <see cref="LookupOrganisationController"/> with the given services.
     /// </summary>
     /// <param name="organisationClient">A client for searching for an organisation in ER.</param>
```

Comment:
Nitpick: Replace the term "searching" with lookup in documentation comments.

Reply:
Haha du er grundig. Veldig nice 👏

Reply:
Frivillig å bry seg om slike nitpicks altså. :P